vent/controller/control_module.py

```diff
@@ -136,8 +139,10 @@ def __init__(self, save_logs: bool = False, flush_every: int = 10):
         self._DATA_dpdt     = 0           # Current sample of the rate of change of pressure dP/dt in cmH2O/sec
         self.__DATA_old     = None
         self._last_update   = time.time()
-        self._flow_list = deque(maxlen = 500)          # An archive of past flows, to calculate background flow out
-        self._DATA_PRESSURE_LIST = list()
+        self._BASELINE_ESTIMATOR_LENGTH = 500
```

Comment:
Some reorganization as discussed: safe raw_flow in the hdf-files, and doing the baseline-subtraction in the main-loop